src/Arcus.Security.Providers.KeyPerFile/Arcus.Security.Providers.KeyPerFile.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
+  </PropertyGroup>
```

Comment:
```suggestion
<PropertyGroup>
    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
    <Authors>Arcus</Authors>
    <Description>Provides support for Docker Secrets</Description>
    <Copyright>Copyright (c) Arcus</Copyright>
    <PackageLicenseUrl>https://github.com/arcus-azure/arcus.security/blob/master/LICENSE</PackageLicenseUrl>
    <PackageProjectUrl>https://github.com/arcus-azure/arcus.security</PackageProjectUrl>
    <PackageIconUrl>https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png</PackageIconUrl>
    <RepositoryUrl>https://github.com/arcus-azure/arcus.security</RepositoryUrl>
    <RepositoryType>Git</RepositoryType>
    <PackageTags>Docker;User Secrets Manager;Security;Key Per FIle</PackageTags>
    <AssemblyName>Arcus.Security.Providers.KeyPerFile</AssemblyName>
    <RootNamespace>Arcus.Security.Providers.KeyPerFile</RootNamespace>
    <PackageId>Arcus.Security.Providers.KeyPerFile</PackageId>
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
  </PropertyGroup>
```

Reply:
Done.  I've chosen some different tasks though instead of the ones you've suggested.